src/Raven.Client/Documents/Session/DocumentsById.cs

```diff
@@ -48,4 +67,9 @@ IEnumerator IEnumerable.GetEnumerator()
             return GetEnumerator();
         }
     }
+
+    public class TrackedEntity
```

Comment:
Let's rename it to `EntityInfo` and add `Id` property. This way it might be more universal class (just in case we want to use / extend it in the future in other place)